cmd/kcoin/chaincmd.go

```diff
@@ -157,8 +159,14 @@ func initGenesis(ctx *cli.Context) error {
 }
 
 //extractKeyCoin returns the keycoin used for launching the client. For now it always return kusd.
-func extractKeyCoin(context *cli.Context) string {
-	return "kusd"
+func extractKeyCoin(ctx *cli.Context) string {
+	kcoin := ctx.GlobalString(utils.CurrencyFlag.Name)
+
+	if kcoin == "" {
```

Comment:
Its default value is `"kusd"`. Why do we need this if? Just for safety?

Reply:
Yeah, in theory never will happen.